gapic-generator/lib/gapic/presenters/service_presenter.rb

```diff
@@ -50,6 +50,10 @@ def package
         PackagePresenter.new @gem_presenter, @api, @service.parent.package
       end
 
+      def is_deprecated?
+        @service.is_deprecated?
```

Comment:
Please add the yard comments with at least the `@return ` here and for other functions